File: fix.py
import sys
with open('src/features/access/access-template.js', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('\\"', '"').replace('\\n', '\n')

with open('src/features/access/access-template.js', 'w', encoding='utf-8') as f:
    f.write(content)
print('Done!')

File: fix.js
const fs = require('fs');
let content = fs.readFileSync('src/features/access/access-template.js', 'utf8');
content = content.replace(/\\"/g, '"').replace(/\\n/g, '\n');
fs.writeFileSync('src/features/access/access-template.js', content);
console.log('Done!');
